docs/releases/1.10.txt

```diff
@@ -1008,3 +1008,5 @@ removed in Django 1.10 (please see the :ref:`deprecation timeline
 
 * The warning that :class:`~django.core.signing.Signer` issued when given an
   invalid separator is now a ``ValueError``.
+
+* The undocumented `xmlhttp` from `core.js` was removed.
```

Comment:
I don't think it needs a mention in the release notes, but this section is for features that went through a deprecation cycle. Also single backticks have no effect on the rendered output, they must be doubled.